lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -349,7 +349,7 @@ impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
         .filter(|&idx| check_deleted_condition(idx, deleted_vectors, deleted_point_bitslice))
         .collect_vec();
 
-        let sparse_vector = self.indices_tracker.remap_vector(sparse_vector.to_owned());
+        let sparse_vector = self.indices_tracker.remap_vector(sparse_vector.clone());
```

Comment:
While I'm fine with this change, I believe it's exactly the same on assembly level.

Reply:
Yes the calls are equivalent.
I prefer calling explicitly `clone` in that case because we do not depend on the `ToOwned` trait abstraction.